src/main/java/mate/academy/internetshop/controller/shoppingcart/ShoppingCartController.java

```diff
@@ -24,6 +24,7 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
         ShoppingCart shoppingCart = shoppingCartService.getByUserId(userId);
         List<Product> products = shoppingCartService.getAllProducts(shoppingCart);
         req.setAttribute("Products", products);
```

Comment:
```suggestion
        req.setAttribute("Products", shoppingCart.getProducts());
```
і краще давай назвемо цей клас GetShoppingCartController щоб було зрозуміліше за що він відповідає

Reply:
+